### DISCUSSION TITLE
Are there any coding conventions when using cute?

### DISCUSSION THREAD

**Discussion by umiswing** [NVIDIA/cutlass#1760] [Q&A]
Recently, I'm developing some kernels using cute. Sometimes I encounter unexpected slow down or computation error (e.g., just adding a template argument which is unused will cause slow down and computation error). I refer to nvcc team, they told me it's caused by bug of nvcc and provide me some workaround. However, I find such unexpected slow down or computation error happens a bit frequently and makes it difficult to add features to existing kernels. Is there any coding conventions I can refer to to avoid such problem when using cute?

**Comment by thakkarV** [comment]
This question is too vague. Without exact code samples or AB diffs, its hard for us to help you.
